src/app/src/index.js

```diff
@@ -17,6 +17,10 @@ import { setGlobalContext, configureContext } from './context';
 const ctx = configureContext(window.REDUX_INITIAL_DATA).cradle;
 setGlobalContext(ctx);
 
+// ctx.store.subscribe(() => {
```

Comment:
@sk91 can't it be remove as it is commented out?

Reply:
@rodrigopavezi fixed